{
  "type": "issue",
  "issue": {
    "id": 1424326437,
    "node_id": "I_kwDOABII585U5X8l",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/26397",
    "number": 26397,
    "state": "closed",
    "state_reason": "completed",
    "title": "Cannot obtain a lock on data directory",
    "body": "I got error while launch bitcoin core 23.0 on macos after I set data loction on my remote NAS,\r\nError: Cannot obtain a lock on data directory /Volumes/Bitcoin Core Data. Bitcoin Core is probably already running.\r\n\r\nIt's still error after I reinstall. And I can not find setting file at ~/Library/Application Support/Bitcoin/",
    "user": {
      "login": "zx0922",
      "id": 12860661,
      "node_id": "MDQ6VXNlcjEyODYwNjYx",
      "avatar_url": "https://avatars.githubusercontent.com/u/12860661?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/zx0922",
      "html_url": "https://github.com/zx0922",
      "followers_url": "https://api.github.com/users/zx0922/followers",
      "following_url": "https://api.github.com/users/zx0922/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/zx0922/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/zx0922/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/zx0922/subscriptions",
      "organizations_url": "https://api.github.com/users/zx0922/orgs",
      "repos_url": "https://api.github.com/users/zx0922/repos",
      "events_url": "https://api.github.com/users/zx0922/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/zx0922/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [],
    "assignees": [],
    "author_association": "NONE",
    "locked": false,
    "comments": 3,
    "closed_at": "2022-10-27T08:42:02Z",
    "created_at": "2022-10-26T16:19:10Z",
    "updated_at": "2022-10-27T08:42:02Z"
  },
  "events": [
    {
      "event": "labeled",
      "id": 7674440680,
      "node_id": "LE_lADOABII585U5X8lzwAAAAHJbqvo",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/7674440680",
      "actor": {
        "login": "zx0922",
        "id": 12860661,
        "node_id": "MDQ6VXNlcjEyODYwNjYx",
        "avatar_url": "https://avatars.githubusercontent.com/u/12860661?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/zx0922",
        "html_url": "https://github.com/zx0922",
        "followers_url": "https://api.github.com/users/zx0922/followers",
        "following_url": "https://api.github.com/users/zx0922/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/zx0922/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/zx0922/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/zx0922/subscriptions",
        "organizations_url": "https://api.github.com/users/zx0922/orgs",
        "repos_url": "https://api.github.com/users/zx0922/repos",
        "events_url": "https://api.github.com/users/zx0922/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/zx0922/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-26T16:19:10Z",
      "label": {
        "name": "Bug",
        "color": "FBBAAB"
      }
    },
    {
      "event": "commented",
      "id": 1292300303,
      "node_id": "IC_kwDOABII585NBvAP",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1292300303",
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-26T16:25:36Z",
      "updated_at": "2022-10-26T16:25:36Z",
      "author_association": "MEMBER",
      "body": "> on macos after I set data loction on my remote NAS,\r\n\r\nI doubt such a setup is supported.",
      "user": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/26397#issuecomment-1292300303",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397"
    },
    {
      "event": "commented",
      "id": 1293101247,
      "node_id": "IC_kwDOABII585NEyi_",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1293101247",
      "actor": {
        "login": "zx0922",
        "id": 12860661,
        "node_id": "MDQ6VXNlcjEyODYwNjYx",
        "avatar_url": "https://avatars.githubusercontent.com/u/12860661?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/zx0922",
        "html_url": "https://github.com/zx0922",
        "followers_url": "https://api.github.com/users/zx0922/followers",
        "following_url": "https://api.github.com/users/zx0922/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/zx0922/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/zx0922/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/zx0922/subscriptions",
        "organizations_url": "https://api.github.com/users/zx0922/orgs",
        "repos_url": "https://api.github.com/users/zx0922/repos",
        "events_url": "https://api.github.com/users/zx0922/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/zx0922/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-27T07:21:51Z",
      "updated_at": "2022-10-27T07:21:51Z",
      "author_association": "NONE",
      "body": "> > on macos after I set data loction on my remote NAS,\r\n> \r\n> I doubt such a setup is supported.\r\n\r\nyes. But I can't set a new path",
      "user": {
        "login": "zx0922",
        "id": 12860661,
        "node_id": "MDQ6VXNlcjEyODYwNjYx",
        "avatar_url": "https://avatars.githubusercontent.com/u/12860661?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/zx0922",
        "html_url": "https://github.com/zx0922",
        "followers_url": "https://api.github.com/users/zx0922/followers",
        "following_url": "https://api.github.com/users/zx0922/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/zx0922/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/zx0922/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/zx0922/subscriptions",
        "organizations_url": "https://api.github.com/users/zx0922/orgs",
        "repos_url": "https://api.github.com/users/zx0922/repos",
        "events_url": "https://api.github.com/users/zx0922/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/zx0922/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/26397#issuecomment-1293101247",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397"
    },
    {
      "event": "commented",
      "id": 1293189409,
      "node_id": "IC_kwDOABII585NFIEh",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1293189409",
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-27T08:41:07Z",
      "updated_at": "2022-10-27T08:41:07Z",
      "author_association": "MEMBER",
      "body": "Try to remove the `.lock` file from your data directory.\r\n\r\nIf the problem still persists, it is related to macOS's `fctnl()` implementation, not to Bitcoin Core code.\r\n\r\nGoing to close this issue.",
      "user": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/26397#issuecomment-1293189409",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26397"
    },
    {
      "event": "unlabeled",
      "id": 7680035484,
      "node_id": "UNLE_lADOABII585U5X8lzwAAAAHJxAqc",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/7680035484",
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-27T08:41:30Z",
      "label": {
        "name": "Bug",
        "color": "FBBAAB"
      }
    },
    {
      "event": "closed",
      "id": 7680040560,
      "node_id": "CE_lADOABII585U5X8lzwAAAAHJxB5w",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/7680040560",
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-27T08:42:03Z"
    },
    {
      "event": "cross-referenced",
      "id": null,
      "node_id": null,
      "url": null,
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-10-28T11:10:36Z",
      "updated_at": "2022-10-28T11:10:36Z",
      "source": {
        "issue": {
          "id": 1425571542,
          "node_id": "I_kwDOABII585U-H7W",
          "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26400",
          "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
          "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26400/labels%7B/name%7D",
          "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26400/comments",
          "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26400/events",
          "html_url": "https://github.com/bitcoin/bitcoin/issues/26400",
          "number": 26400,
          "state": "closed",
          "state_reason": "completed",
          "title": "v22.0 wallet freezes on Mac OS Ventura on arm64",
          "body": "I upgraded to the newest Mac OS Ventura. The Bitcoin Core application was deleted automatically from my computer. I downloaded Bitcoin core again fro my Mac. and when I tried to open Bitcoin core it has been stuck on Verifying Wallet(s) for over an hour. \r\n\r\n<img width=\"476\" alt=\"Screenshot 2022-10-27 at 08 39 24\" src=\"https://user-images.githubusercontent.com/116812829/198286720-66e119e2-1c2b-453d-928c-9b4abc794240.png\">\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n<!-- This issue tracker is only for technical issues related to Bitcoin Core.\r\n\r\nGeneral bitcoin questions and/or support requests are best directed to the Bitcoin StackExchange at https://bitcoin.stackexchange.com.\r\n\r\nFor reporting security issues, please read instructions at https://bitcoincore.org/en/contact/.\r\n\r\nIf the node is \"stuck\" during sync or giving \"block checksum mismatch\" errors, please ensure your hardware is stable by running memtest and observe CPU temperature with a load-test tool such as linpack before creating an issue! -->\r\n\r\n<!-- Describe the issue -->\r\n\r\n**Expected behavior**\r\n\r\n<!--- What behavior did you expect? -->\r\n\r\n**Actual behavior**\r\n\r\n<!--- What was the actual behavior (provide screenshots if the issue is GUI-related)? -->\r\n\r\n**To reproduce**\r\n\r\n<!--- How reliably can you reproduce the issue, what are the steps to do so? -->\r\n\r\n**System information**\r\n\r\n<!-- What version of Bitcoin Core are you using, where did you get it (website, self-compiled, etc)? -->\r\n\r\n<!-- What type of machine are you observing the error on (OS/CPU and disk type)? -->\r\n\r\n<!-- GUI-related issue? What is your operating system and its version? If Linux, what is your desktop environment and graphical shell? -->\r\n\r\n<!-- Any extra information that might be useful in the debugging process. -->\r\n<!--- This is normally the contents of a `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred. -->\r\n",
          "user": {
            "login": "cryptodc1",
            "id": 116812829,
            "node_id": "U_kgDOBvZsHQ",
            "avatar_url": "https://avatars.githubusercontent.com/u/116812829?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/cryptodc1",
            "html_url": "https://github.com/cryptodc1",
            "followers_url": "https://api.github.com/users/cryptodc1/followers",
            "following_url": "https://api.github.com/users/cryptodc1/following%7B/other_user%7D",
            "gists_url": "https://api.github.com/users/cryptodc1/gists%7B/gist_id%7D",
            "starred_url": "https://api.github.com/users/cryptodc1/starred%7B/owner%7D%7B/repo%7D",
            "subscriptions_url": "https://api.github.com/users/cryptodc1/subscriptions",
            "organizations_url": "https://api.github.com/users/cryptodc1/orgs",
            "repos_url": "https://api.github.com/users/cryptodc1/repos",
            "events_url": "https://api.github.com/users/cryptodc1/events%7B/privacy%7D",
            "received_events_url": "https://api.github.com/users/cryptodc1/received_events",
            "type": "User",
            "site_admin": false
          },
          "labels": [
            {
              "id": 64585,
              "node_id": "MDU6TGFiZWw2NDU4NQ==",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Bug",
              "name": "Bug",
              "color": "FBBAAB",
              "default": false
            },
            {
              "id": 149424,
              "node_id": "MDU6TGFiZWwxNDk0MjQ=",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Wallet",
              "name": "Wallet",
              "color": "08a781",
              "default": false
            },
            {
              "id": 234879,
              "node_id": "MDU6TGFiZWwyMzQ4Nzk=",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/macOS",
              "name": "macOS",
              "description": "",
              "color": "660000",
              "default": false
            }
          ],
          "assignees": [],
          "author_association": "NONE",
          "locked": false,
          "comments": 30,
          "closed_at": "2022-10-28T13:58:24Z",
          "created_at": "2022-10-27T12:40:32Z",
          "updated_at": "2022-10-28T13:58:24Z"
        },
        "type": "issue"
      }
    },
    {
      "event": "cross-referenced",
      "id": null,
      "node_id": null,
      "url": null,
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-11-12T10:33:15Z",
      "updated_at": "2022-11-12T10:33:15Z",
      "source": {
        "issue": {
          "id": 1436654238,
          "node_id": "I_kwDOABII585VoZqe",
          "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26455",
          "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
          "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26455/labels%7B/name%7D",
          "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26455/comments",
          "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26455/events",
          "html_url": "https://github.com/bitcoin/bitcoin/issues/26455",
          "number": 26455,
          "state": "closed",
          "state_reason": "completed",
          "title": "Flushing block file to disk failed when using external drive on macOS Ventura",
          "body": "I kept getting an error regarding Flushing Block to disk. I've tried to use bitcoin core V0.23 and it doesn't help; although V0.22 was better at loading the backup file, but I couldn't send any of my bitcoin out. I can't seem to load the wallet or the backup file properly. Not a coder and hoping someone can help. I don't think it's an issue with the hard drive. Any suggestion? I'm on MacOSx Ventura 13.0. \r\n\r\n\r\n\r\nDebug.log file:\r\n\r\n2022-11-04T20:34:28Z Using wallet /Volumes/Extreme SSD/bruceltd wallet.dat\r\n2022-11-04T20:34:28Z BerkeleyEnvironment::Open: LogDir=/Volumes/Extreme SSD/database ErrorFile=/Volumes/Extreme SSD/db.log\r\n.\r\n.\r\n.\r\n2022-11-04T20:58:16Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-04T20:58:16Z ERROR: SerializeFileDB: Failed to flush file /Volumes/Extreme SSD/peers.176f\r\n\r\ndb.log file:\r\nfsync: Inappropriate ioctl for device\r\nPANIC: Inappropriate ioctl for device\r\ntxn_checkpoint: failed to flush the buffer cache: DB_RUNRECOVERY: Fatal error, run database recovery\r\n\r\n\r\n",
          "user": {
            "login": "aureliusltd",
            "id": 117485255,
            "node_id": "U_kgDOBwCuxw",
            "avatar_url": "https://avatars.githubusercontent.com/u/117485255?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/aureliusltd",
            "html_url": "https://github.com/aureliusltd",
            "followers_url": "https://api.github.com/users/aureliusltd/followers",
            "following_url": "https://api.github.com/users/aureliusltd/following%7B/other_user%7D",
            "gists_url": "https://api.github.com/users/aureliusltd/gists%7B/gist_id%7D",
            "starred_url": "https://api.github.com/users/aureliusltd/starred%7B/owner%7D%7B/repo%7D",
            "subscriptions_url": "https://api.github.com/users/aureliusltd/subscriptions",
            "organizations_url": "https://api.github.com/users/aureliusltd/orgs",
            "repos_url": "https://api.github.com/users/aureliusltd/repos",
            "events_url": "https://api.github.com/users/aureliusltd/events%7B/privacy%7D",
            "received_events_url": "https://api.github.com/users/aureliusltd/received_events",
            "type": "User",
            "site_admin": false
          },
          "labels": [
            {
              "id": 234879,
              "node_id": "MDU6TGFiZWwyMzQ4Nzk=",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/macOS",
              "name": "macOS",
              "description": "",
              "color": "660000",
              "default": false
            },
            {
              "id": 97470796,
              "node_id": "MDU6TGFiZWw5NzQ3MDc5Ng==",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/UTXO%20Db%20and%20Indexes",
              "name": "UTXO Db and Indexes",
              "color": "fbca04",
              "default": false
            },
            {
              "id": 323697996,
              "node_id": "MDU6TGFiZWwzMjM2OTc5OTY=",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Data%20corruption",
              "name": "Data corruption",
              "color": "f7c6c7",
              "default": false
            },
            {
              "id": 477890092,
              "node_id": "MDU6TGFiZWw0Nzc4OTAwOTI=",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Questions%20and%20Help",
              "name": "Questions and Help",
              "color": "006688",
              "default": false
            }
          ],
          "assignees": [],
          "author_association": "NONE",
          "locked": false,
          "comments": 12,
          "closed_at": "2022-11-30T09:59:39Z",
          "created_at": "2022-11-04T21:14:06Z",
          "updated_at": "2022-12-07T09:10:30Z"
        },
        "type": "issue"
      }
    },
    {
      "event": "cross-referenced",
      "id": null,
      "node_id": null,
      "url": null,
      "actor": {
        "login": "hebasto",
        "id": 32963518,
        "node_id": "MDQ6VXNlcjMyOTYzNTE4",
        "avatar_url": "https://avatars.githubusercontent.com/u/32963518?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hebasto",
        "html_url": "https://github.com/hebasto",
        "followers_url": "https://api.github.com/users/hebasto/followers",
        "following_url": "https://api.github.com/users/hebasto/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/hebasto/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/hebasto/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/hebasto/subscriptions",
        "organizations_url": "https://api.github.com/users/hebasto/orgs",
        "repos_url": "https://api.github.com/users/hebasto/repos",
        "events_url": "https://api.github.com/users/hebasto/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/hebasto/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2022-11-18T15:03:22Z",
      "updated_at": "2022-11-18T15:03:22Z",
      "source": {
        "issue": {
          "id": 1455183006,
          "node_id": "I_kwDOABII585WvFSe",
          "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26530",
          "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
          "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26530/labels%7B/name%7D",
          "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26530/comments",
          "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/26530/events",
          "html_url": "https://github.com/bitcoin/bitcoin/issues/26530",
          "number": 26530,
          "state": "closed",
          "state_reason": "completed",
          "title": "Breking in Mac OS",
          "body": "I'm trying to use bitcoin core on mac OS\r\nI want to record on an external hd\r\n\r\nbut i get the following error all the time\r\n\r\n2022-11-17T22:07:44Z msghand thread exit\r\n2022-11-17T22:07:44Z net thread exit\r\n2022-11-17T22:07:44Z opencon thread exit\r\n2022-11-17T22:07:44Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:44Z ERROR: SerializeFileDB: Failed to flush file /Volumes/Extreme SSD/bitcoin_core/peers.8611\r\n2022-11-17T22:07:44Z DumpAnchors: Flush 2 outbound block-relay-only peer addresses to anchors.dat started\r\n2022-11-17T22:07:44Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:44Z ERROR: SerializeFileDB: Failed to flush file /Volumes/Extreme SSD/bitcoin_core/anchors.ceb3\r\n2022-11-17T22:07:44Z DumpAnchors: Flush 2 outbound block-relay-only peer addresses to anchors.dat completed (0.01s)\r\n2022-11-17T22:07:44Z scheduler thread exit\r\n2022-11-17T22:07:44Z Writing 0 unbroadcast transactions to disk.\r\n2022-11-17T22:07:44Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:44Z Failed to dump mempool: FileCommit failed. Continuing anyway.\r\n2022-11-17T22:07:45Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:45Z ERROR: Flush: failed to commit file 1\r\n2022-11-17T22:07:45Z *** Flushing block file to disk failed. This is likely the result of an I/O error.\r\n2022-11-17T22:07:45Z Error: A fatal internal error occurred, see debug.log for details\r\n2022-11-17T22:07:45Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:45Z ERROR: Flush: failed to commit file 1\r\n2022-11-17T22:07:45Z *** Flushing undo file to disk failed. This is likely the result of an I/O error.\r\n2022-11-17T22:07:45Z Error: A fatal internal error occurred, see debug.log for details\r\n2022-11-17T22:07:46Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:46Z ERROR: Flush: failed to commit file 1\r\n2022-11-17T22:07:46Z *** Flushing block file to disk failed. This is likely the result of an I/O error.\r\n2022-11-17T22:07:46Z Error: A fatal internal error occurred, see debug.log for details\r\n2022-11-17T22:07:46Z FileCommit: fcntl F_FULLFSYNC failed: 25\r\n2022-11-17T22:07:46Z ERROR: Flush: failed to commit file 1\r\n2022-11-17T22:07:46Z *** Flushing undo file to disk failed. This is likely the result of an I/O error.\r\n2022-11-17T22:07:46Z Error: A fatal internal error occurred, see debug.log for details\r\n2022-11-17T22:07:46Z Shutdown: done\r\n",
          "user": {
            "login": "mateusgalasso",
            "id": 3990028,
            "node_id": "MDQ6VXNlcjM5OTAwMjg=",
            "avatar_url": "https://avatars.githubusercontent.com/u/3990028?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/mateusgalasso",
            "html_url": "https://github.com/mateusgalasso",
            "followers_url": "https://api.github.com/users/mateusgalasso/followers",
            "following_url": "https://api.github.com/users/mateusgalasso/following%7B/other_user%7D",
            "gists_url": "https://api.github.com/users/mateusgalasso/gists%7B/gist_id%7D",
            "starred_url": "https://api.github.com/users/mateusgalasso/starred%7B/owner%7D%7B/repo%7D",
            "subscriptions_url": "https://api.github.com/users/mateusgalasso/subscriptions",
            "organizations_url": "https://api.github.com/users/mateusgalasso/orgs",
            "repos_url": "https://api.github.com/users/mateusgalasso/repos",
            "events_url": "https://api.github.com/users/mateusgalasso/events%7B/privacy%7D",
            "received_events_url": "https://api.github.com/users/mateusgalasso/received_events",
            "type": "User",
            "site_admin": false
          },
          "labels": [
            {
              "id": 64585,
              "node_id": "MDU6TGFiZWw2NDU4NQ==",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Bug",
              "name": "Bug",
              "color": "FBBAAB",
              "default": false
            },
            {
              "id": 234879,
              "node_id": "MDU6TGFiZWwyMzQ4Nzk=",
              "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/macOS",
              "name": "macOS",
              "description": "",
              "color": "660000",
              "default": false
            }
          ],
          "assignees": [],
          "author_association": "NONE",
          "locked": true,
          "comments": 3,
          "closed_at": "2022-11-18T15:14:25Z",
          "created_at": "2022-11-18T14:05:34Z",
          "updated_at": "2022-11-22T16:41:10Z"
        },
        "type": "issue"
      }
    }
  ]
}